app/Resources/translations/custom_tags.en.yaml

```diff
@@ -14,3 +14,27 @@ ezrichtext.custom_tags.ezyoutube.attributes.height.label: Height
 ezrichtext.custom_tags.ezyoutube.attributes.title.label: Title
 ezrichtext.custom_tags.ezyoutube.attributes.video_url.label: 'Video url'
 ezrichtext.custom_tags.ezyoutube.attributes.width.label: Width
+
+# eztwitter
+ezrichtext.custom_tags.eztwitter.label: Twitter
+ezrichtext.custom_tags.eztwitter.description: ''
+ezrichtext.custom_tags.eztwitter.attributes.cards.label: Cards
+ezrichtext.custom_tags.eztwitter.attributes.conversation.label: Conversation
+ezrichtext.custom_tags.eztwitter.attributes.dnt.label: DNT
+ezrichtext.custom_tags.eztwitter.attributes.lang.label: Lang
+ezrichtext.custom_tags.eztwitter.attributes.link_color.label: 'Link color'
+ezrichtext.custom_tags.eztwitter.attributes.theme.label: Theme
+ezrichtext.custom_tags.eztwitter.attributes.tweet_url.label: 'Tweet url'
+ezrichtext.custom_tags.eztwitter.attributes.width.label: Width
+
+# factbox
+ezrichtext.custom_tags.factbox.label: Factbox
+ezrichtext.custom_tags.factbox.description: ''
+ezrichtext.custom_tags.factbox.attributes.title.label: Title# ezyoutube
+ezrichtext.custom_tags.ezyoutube.label: YouTube
```

Comment:
I think this part can be removed. Youtube custom tag is already defined here: https://github.com/ezsystems/ezplatform-com/pull/161/files#diff-6b38605c4958f1581d51bf442b518837R9

Reply:
vim...